docs/src/index.md

```diff
@@ -109,7 +109,7 @@ Attribute   | Type               | Notes
 `nnzj`      | `Int             ` | number of nonzeros in the sparse Jacobian
 `lin_nnzj`  | `Int             ` | number of nonzeros in the sparse linear constraints Jacobian
 `nln_nnzj`  | `Int             ` | number of nonzeros in the sparse nonlinear constraints Jacobian
-`nnzh`      | `Int             ` | number of nonzeros in the sparse Hessian
+`nnzh`      | `Int             ` | number of nonzeros in the sparse Hessian of the Lagrangian
```

Comment:
It's actually even, the number of nonzeros in the lower triangle of the sparse Hessian of the Lagrangian :)

Reply:
You're right and I forgot to talk about this triangle when I explained it to Caleb!